android/4_build_android_port.sh

```diff
@@ -32,14 +32,14 @@ git clone https://github.com/Ghabry/timidity_gus.git assets/timidity
 android update project --path "." --target 1
 
 # Build
-ndk-build -j4 NDK_LIBS_OUT=./jniLibs
```

Comment:
oh its linux specific?
i also found this:
http://stackoverflow.com/questions/2527496/how-can-i-write-a-makefile-to-auto-detect-and-parallelize-the-build-with-gnu-mak
http://stackoverflow.com/questions/2527496/how-can-i-write-a-makefile-to-auto-detect-and-parallelize-the-build-with-gnu-mak